proto/ecosystem-incentive/event.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto3";
+package ununifi.ecosystemincentive;
+
+import "ecosystem-incentive/ecosystem_incentive.proto";
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/UnUniFi/chain/x/ecosystem-incentive/types";
+
+message EventRegister {
+  string incentive_unit_id = 1 [ (gogoproto.moretags) = "yaml:\"incentive_unit_id\"" ];
+  repeated SubjectInfo subject_info_list = 2 [
+    (gogoproto.moretags) = "yaml:\"subject_info_lists\"",
+    (gogoproto.nullable) = false
+  ];
+}
+
+message EventWithdrawAllRewards {
+  string sender = 1 [
+    (gogoproto.moretags) = "yaml:\"sender\"",
+    (gogoproto.customtype) = "github.com/UnUniFi/chain/types.StringAccAddress",
+    (gogoproto.nullable) = false
+  ];
+  repeated cosmos.base.v1beta1.Coin rewards = 2 [
+    (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins",
+    (gogoproto.moretags) = "yaml:\"all_withdrawn_rewards\"",
```

Comment:
more tag and variable name mismatch. rewards and all_withdrawn_rewards

Reply:
modified it.